scripts/reset_stale_workers.py

```diff
@@ -85,14 +85,20 @@ def main(arguments):
         logger.info("Invalid project id")
         sys.exit(0)
 
-    # Attach timezone to naive date value
+    # First check if relative date
+    logger.info("Formatting date")
     try:
-        local_cutoff_date = pendulum.from_format(arguments.cutoff_date, "MM/DD/YYYY hh:mm:ss", tz=args.timezone, formatter='alternative')
-    except Exception as e:
-        logger.info(e)
-        logger.info("Invalid date format. Please check documentation and try again")
-        sys.exit(0)
-    utc_dt = local_cutoff_date.in_tz('UTC')
+        delta = int(args.cutoff_date)
+        utc_dt = pendulum.now().subtract(minutes=delta).in_tz('UTC')
+    except Exception:
```

Comment:
Might want to narrow the exception here to some Pendulum exception.

Reply:
I also need to check if the cutoff_date is an int here